spec/components/email_updater_spec.rb

```diff
@@ -17,6 +17,92 @@
     expect(updater.errors.messages[:base].first).to be I18n.t("change_email.error_staged")
   end
 
+  context "when an admin is changing the email of a user" do
+    let(:email_change_requesting_user) { Fabricate(:admin) }
+    let(:updater) { EmailUpdater.new(user.guardian, user, email_change_requesting_user) }
+
+    def expect_old_email_job
+      Jobs.expects(:enqueue).with(:critical_user_email, has_entries(to_address: "old.email@example.com", type: :notify_old_email, user_id: user.id))
```

Comment:
Might be better to get rid of the string when we already have it in a variable?

```suggestion
      Jobs.expects(:enqueue).with(:critical_user_email, has_entries(to_address: old_email, type: :notify_old_email, user_id: user.id))
```